tests/cases/case_helpers_care_platform.yaml

```diff
@@ -0,0 +1,55 @@
+care_platform_logout:
+  Environment: Settings 
+  Roles:
+    - Role: desktopChrome
+      App: desktop       
+  Actions:     
+      # Login to the Care Platform app with Care Partners account
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_header.user_dropdown_button$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_header.logout_menu_button$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_logout_modal.stay_on_shift_button$
```

Comment:
Can we use this as a parameter, so we can do the log out with the three options: "Yes, Go Off Shift", "No, Stay On Shift" and "Cancel Logout"?

Reply:
yes - solved